postgresql-async/src/main/java/com/github/jasync/sql/db/postgresql/exceptions/MissingCredentialInformationException.kt

```diff
@@ -1,16 +1,10 @@
 package com.github.jasync.sql.db.postgresql.exceptions
 
 import com.github.jasync.sql.db.exceptions.DatabaseException
-import com.github.jasync.sql.db.postgresql.messages.backend.AuthenticationResponseType
 
 class MissingCredentialInformationException(
     val username: String,
-    val password: String?,
-    private val authenticationResponseType: AuthenticationResponseType
+    val password: String?
 ) : DatabaseException(
-    "Username and password were required by auth type %s but are not available (username=<%s> password=<%s>".format(
-        authenticationResponseType,
-        username,
-        password
-    )
+    "Username and password were required but are not available (username=<$username> password=<$password>)"
```

Comment:
I know it was like that before, but maybe it's better not to put the password to the message? (maybe only first two chars?)

Reply:
Completely agree. I've pushed a fix that leaves the password out entirely while still giving an indication if a password was provided.